Sprint-1/implement/max.js

```diff
@@ -1,4 +1,33 @@
 function findMax(elements) {
-}
+    // return -Infinity if elements is empty. If elements has 1 item, return this item.
+    if (elements.length === 0) {
+        return -Infinity;
+    } else if (elements.length === 1 && typeof elements[elements.length - 1] == "number") {
```

Comment:
I completely agree with you. The code for handling an array of size 1 isn't needed because the rest of the code already works for that case, making it simpler and shorter.